website/src/docs/google-drive.md

```diff
@@ -39,6 +39,37 @@ In the [CDN package](/docs/#With-a-script-tag), it is available on the `Uppy` gl
 const GoogleDrive = Uppy.GoogleDrive
 ```
 
+## Setting  Up
+
+To use the Google Drive provider, you need to configure the Google Drive keys that Companion should use. With the standalone Companion server, specify environment variables:
+```shell
+export COMPANION_GOOGLE_KEY="Google Drive OAuth client ID"
+export COMPANION_GOOGLE_SECRET="Google Drive OAuth client secret"
+```
+
+When using the Companion Node.js API, configure these options:
+```js
+companion.app({
+  providerOptions: {
+    drive: {
+      key: 'Google Drive OAuth client ID',
+      secret: 'Google Drive OAuth client secret'
+    }
+  }
+})
+```
+
+To sign up for API keys, go to the [Google Developer Console](https://console.developers.google.com/).
+
+Create a project for your app if you don't have one yet.
+- On the project's dashboard, [enable the Google Drive API](https://developers.google.com/drive/api/v3/enable-drive-api).
+- [Set up OAuth authorization](https://developers.google.com/drive/api/v3/about-auth). Use this for an authorized redirect URI:
+   ```
+   https://$YOUR_COMPANION_HOST_NAME/connect/drive/callback
```

Comment:
```suggestion
   https://$YOUR_COMPANION_HOST_NAME/connect/google/callback
```
Another inconsistency that we should fix as per [this](https://github.com/transloadit/uppy/pull/2346#issuecomment-651096963)

Reply:
isn't the callback URL already `/connect/drive/callback`? 🤔

Reply:
so the current flow is `grant` makes oauth calls **->** grant defaults redirect_uri to internal values (in this case ) /connect/google/callback **->** grant receives redirect request, does some processing, then forwards required data to companion's callback `/drive/callback` **->** Companion receives the callback at `/drive/callback` and sends the required dat to the client.

So yeah, it's quite confusing how it works now. Inconsistencies because grant's defaults are allowed, and its options are also bubbled up to the public interface.